File: import/python/import.py
from neo4j.v1 import GraphDatabase, basic_auth
import sys

password = "neo"

query = """\
MERGE (p1:Person {name: {person1} })
MERGE (p2:Person {name: {person2} })
MERGE (p1)-[:KNOWS]->(p2)
"""

def import_data(person1, person2):
    print("Creating relationship between {p1} and {p2}".format(p1=person1, p2=person2))
    with GraphDatabase.driver("bolt://localhost:7687", auth=basic_auth("neo4j", password)) as driver:
        with driver.session() as session:
            session.run(query, {"person1": person1, "person2": person2})

if __name__ == "__main__":
    if len(sys.argv) < 3:
        print("Usage: python import.py [Person1] [Person2]")
        sys.exit(1)

    import_data(sys.argv[1], sys.argv[2])

File: import/javascript/import.js
const neo4j = require('neo4j-driver').v1;
const password = "neo";

const driver = neo4j.driver("bolt://localhost:7687", neo4j.auth.basic("neo4j", password));

const query = `
MERGE (p1:Person {name: {person1} })
MERGE (p2:Person {name: {person2} })
MERGE (p1)-[:KNOWS]->(p2)
`

function importData(person1, person2) {
    const session = driver.session();
    const resultPromise = session.run(query, {person1: person1, person2: person2} );
    resultPromise.then(result => {
      session.close();

      // on application exit:
      driver.close();
    });
}

if(process.argv.length < 4) {
    console.log("Usage: node import.js [Person1] [Person2]")
    process.exit(1);
} else {
    importData(process.argv[2], process.argv[3])
}
